File: src/gestorAplicacion/Sede.java
package gestorAplicacion;

public class Sede {
private int numsede;
private boolean open;
//para saber si la sede esta abierta en el momento
private boolean disp;
//disponibilidad de la sede (si hay mesas libres o no)
private String ubicacion;
private int Cantidademp;
//cantidad de empleados activos en la sede

public Sede(int numsede, boolean open, boolean disp, String ubicacion, int cantidademp) {
	super();
	this.numsede = numsede;
	this.open = open;
	this.disp = disp;
	this.ubicacion = ubicacion;
	Cantidademp = cantidademp;
}

public int getNumsede() {
	return numsede;
}

public void setNumsede(int numsede) {
	this.numsede = numsede;
}

public boolean isOpen() {
	return open;
}

public void setOpen(boolean open) {
	this.open = open;
}

public boolean isDisp() {
	return disp;
}

public void setDisp(boolean disp) {
	this.disp = disp;
}

public String getUbicacion() {
	return ubicacion;
}

public void setUbicacion(String ubicacion) {
	this.ubicacion = ubicacion;
}

public int getCantidademp() {
	return Cantidademp;
}

public void setCantidademp(int cantidademp) {
	Cantidademp = cantidademp;
}


}

File: Python/src/gestorAplicacion/sede.py
class Sede:

    def __init__(self, numsede,open,disp,ubicacion, cantidademp):
        self.numsede = numsede
        self.open=open
        self.disp=disp
        self.ubicacion=ubicacion
        self.cantidademp=cantidademp

    #getters y setters   
    def setNumsede(self, numsedes):
        self.numsede = numsedes
    
    def getNumsede(self):
        return self.numsede

    def setOpen(self, open):
        self.open=open
    
    def isOpen(self):
        return self.open

    def setDisp(self, disponibles):
        self.disp=disponibles
    
    def isDisp(self):
        return self.disp   

    def setUbicacion(self, ubicaciones):
        self.ubicacion=ubicaciones
    
    def getUbicacion(self):
        return self.ubicacion

    def setCantidademp(self, cantidades):
        self.cantidademp=cantidades
    
    def getCantidademp(self):
        return self.cantidademp





	    


   


        